src/lib/gundb/UserStorage.js

```diff
@@ -771,16 +771,16 @@ export class UserStorage {
         .get(address)
         .get('profile')
 
-      avatar =
-        (await profileToShow
-          .get('avatar')
-          .get('display')
-          .then()) || undefined
       fullName =
         (await profileToShow
           .get('fullName')
           .get('display')
           .then()) || (address === '0x0000000000000000000000000000000000000000' ? 'GoodDollar' : address)
+      avatar =
+        (await profileToShow
+          .get('avatar')
+          .get('display')
+          .then()) || (fullName === 'GoodDollar' ? `${process.env.PUBLIC_URL}/favicon-96x96.png` : undefined)
```

Comment:
```suggestion
          .then()) || (address === '0x0000000000000000000000000000000000000000' ? `${process.env.PUBLIC_URL}/favicon-96x96.png` : undefined)
```

Seems better to me to use  `address` since this seems to be the fixed condition. GoodDollar name may change.

Reply:
@dminones @kbari this actually incorrect since GoodDollar is used on "unknown" 0x0 is the "null" address. GoodDollar address is basically the addresses of our smart contracts